src/main/webapp/i18n/de/course.json

```diff
@@ -13,7 +13,26 @@
             "delete": {
                 "question": "Soll der Kurs <strong>{{ title }}</strong> wirklich dauerhaft gelöscht werden? Alle zugehörigen Elemente, inklusive der Klausuren, Aufgaben sowie Repositories und Build-Pläne werden gelöscht. Diese Aktion kann NICHT rückgängig gemacht werden!",
                 "typeNameToConfirm": "Bitte gib den Namen des Kurses zur Bestätigung ein.",
-                "icon": "Kursicon löschen"
+                "icon": "Kursicon löschen",
+                "summary": {
+                    "title": "Kurs Zusammenfassung",
+                    "numberExams": "Anzahl Klausuren",
+                    "numberLectures": "Anzahl Vorlesungen",
+                    "numberStudents": "Anzahl Studierende",
+                    "numberTutors": "Anzahl Tutor:innen",
+                    "numberEditors": "Anzahl Editor:innen",
+                    "numberInstructors": "Anzahl Lehrende",
+                    "numberProgrammingExercises": "Anzahl Programmieraufgaben",
+                    "numberModelingExercises": "Anzahl Modellierungsaufgaben",
+                    "numberTextExercises": "Anzahl Textaufgaben",
+                    "numberFileUploadExercises": "Anzahl Dateiupload-Aufgaben",
+                    "numberQuizExercises": "Anzahl Quizaufgaben",
+                    "numberRepositories": "Anzahl Repositories",
+                    "numberBuilds": "Anzahl Builds",
+                    "numberCommunicationPosts": "Anzahl Kommunikationsbeiträge",
+                    "numberAnswerPosts": "Anzahl Antwortbeiträge",
+                    "isTestCourse": "Test Course"
+                }
```

Comment:
Why are we not using articles for the german translation? Was this an active decision? 
I think we should use articles.

Also the translation "Test Course" to "Test Course" in German does not seem accurate to me.

`Kurs Zusammenfassung` is not separated in german - the correct translation would be `Kurszusammenfassung`

```suggestion
                "summary": {
                    "title": "Kurszusammenfassung",
                    "numberExams": "Anzahl der Klausuren",
                    "numberLectures": "Anzahl der Vorlesungen",
                    "numberStudents": "Anzahl der Studierenden",
                    "numberTutors": "Anzahl der Tutor:innen",
                    "numberEditors": "Anzahl der Editor:innen",
                    "numberInstructors": "Anzahl der Lehrenden",
                    "numberProgrammingExercises": "Anzahl der Programmieraufgaben",
                    "numberModelingExercises": "Anzahl der Modellierungsaufgaben",
                    "numberTextExercises": "Anzahl der Textaufgaben",
                    "numberFileUploadExercises": "Anzahl der Dateiupload-Aufgaben",
                    "numberQuizExercises": "Anzahl der Quizaufgaben",
                    "numberRepositories": "Anzahl der Repositories",
                    "numberBuilds": "Anzahl der Builds",
                    "numberCommunicationPosts": "Anzahl der Kommunikationsbeiträge",
                    "numberAnswerPosts": "Anzahl der Antwortbeiträge",
                    "isTestCourse": "Testkurs"
                }
```